plugins/woocommerce/src/main/kotlin/org/wordpress/android/fluxc/store/WCTaxStore.kt

```diff
@@ -44,20 +58,45 @@ class WCTaxStore @Inject constructor(
                     WCTaxSqlUtils.insertOrUpdateTaxClasses(taxClassModels)
                     WooResult(taxClassModels)
                 }
+
                 else -> WooResult(WooError(GENERIC_ERROR, UNKNOWN))
             }
         }
     }
+
+    // Returns a boolean indicating whether more Tax Rates can be fetched
```

Comment:
```suggestion
    /**
     * returns a boolean indicating whether more Tax Rates can be fetched.
     */
```
It's better to use official kdoc standard for documenting the code.

Reply:
updated https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2852/commits/3cdacc3e21a7fe478de5ec10716630b43e3f9c13